rustyms/src/shared/glycan.rs

```diff
@@ -568,10 +625,24 @@ pub enum HeptoseIsomer {
     Sedoheptulose,
 }
 
+/// Any 9 carbon glycan, these isomers are modification specific (need the correct substituents applied to be meaningful)
+#[allow(dead_code)]
+#[derive(Clone, Eq, PartialEq, Ord, PartialOrd, Hash, Debug, Serialize, Deserialize)]
+pub enum NonoseIsomer {
```

Comment:
This kind of classification is imprecise. What if you have a Kdn with an additional modification or substitution. Does it cease to be `Kdn` or is it `Kdn` + something? Admittedly, these are special cases, but they are only "special" in so far as IUPAC trivial names are concerned.

Reply:
Similarly, `NeuAc` is actually `2-keto-3-deoxy-Nonp5N` with an acidic modification at the first position, plus an `acetyl` added to the amine group at the fifth backbone position. Additionally, `Neu5Ac8Ac` only adds another `acetyl`, not another acetylosamine. 

I haven't used the formal names in ~5 years so the precise spelling could be off.

Reply:
Yes it is imprecise, I only use these to when reading in a short iupac name be able to store the info to tell apart Kdn from a regular Sia and Pse/Leg/Aci from each other. This was needed to render the correct colour for these monosaccharides as currently not enough isomeric state is tracked to tell these apart. I will update the docs to reflect this fact better.
```suggestion
/// Any 9 carbon glycan, these isomers are modification specific (need the correct substituents
/// applied to be meaningful). These are to be used only to store isomeric state that was inferred
/// from other sources that cannot be tracked in other ways in the current structure. Any isomer
/// used that does not have the correct monosaccharide substituents applied is meaningless.
```